.github/workflows/backend.yaml

```diff
@@ -31,5 +31,7 @@ jobs:
         run: docker compose up -d --build db api
       - name: Check for missing migrations
         run: docker compose exec -T api ./manage.py makemigrations --check --dry-run --no-input
+      - name: Install pytest-github-actions-annotate-failures
+        run: docker compose run --rm -T api pip install pytest-github-actions-annotate-failures
```

Comment:
i'm not :100: convinced this will work, maybe you also need to ensure that the GITHUB_TOKEN env var gets passed into the container where the actual tests are run.

Reply:
You are probably right, it didn't annotate any of the ~50 warnings